src/backend/login/usingPassport.js

```diff
@@ -1,36 +1,36 @@
-const passport = require('passport');
```

Comment:
I don't think this file is in the right place.  I'll let @miggs125 chime in, but for sure it needs to be somewhere under ` src/backend/web/*`

Reply:
It depends on how the project is organized, as this only pertains to logging in, I felt it was better to have a seperate folder to keep problems. Isolated, however I can move it if you think it's better organized a different way?